index.html

```diff
@@ -9,20 +9,21 @@
   <header>
     <nav>
       <ul>
-        <li><a href="#">Home</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Services</a></li>
-        <li><a href="#">Contact</a></li>
+        <li><a href="#">• Contact</a></li>
```

Comment:
```suggestion
        <li><a href="#">Contact</a></li>
```
Não faz parte do layout, remover os demais.